### PR TITLE
Fix ts24

### PR DESCRIPTION
**Pull request by tbosch**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: npm-shrinkwrap.clean.json [hunk 1]
```diff
@@ -1,12 +1,18 @@
 {
   "name": "angular-srcs",
-  "version": "5.0.0-beta.5",
+  "version": "5.0.0-beta.6",
   "dependencies": {
     "@bazel/typescript": {
-      "version": "0.0.9",
+      "version": "0.0.7",
```
Comment (tbosch) [inline npm-shrinkwrap.clean.json]: Why this downgrade?

File: npm-shrinkwrap.clean.json [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
       "dependencies": {
         "@types/node": {
           "version": "7.0.18"
+        },
+        "tsickle": {
+          "version": "0.23.6"
+        },
+        "typescript": {
+          "version": "2.3.4"
```
Comment (tbosch) [inline npm-shrinkwrap.clean.json]: This should be 2.4?

File: npm-shrinkwrap.clean.json [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
   "dependencies": {
     "@bazel/typescript": {
-      "version": "0.0.9",
+      "version": "0.0.7",
       "dependencies": {
         "@types/node": {
           "version": "7.0.18"
+        },
+        "tsickle": {
+          "version": "0.23.6"
```
Comment (tbosch) [inline npm-shrinkwrap.clean.json]: This should be 0.24.0 as well (see below)

File: npm-shrinkwrap.clean.json (hunk 1 above)
Comment (tbosch) [reply npm-shrinkwrap.clean.json]: This is in angular-srcs, why do we need this?

File: scripts/ci/install.sh [hunk 4]
```diff
@@ -42,7 +42,7 @@ travisFoldEnd "npm-install"
 
 # Install Selenium WebDriver
 travisFoldStart "webdriver-manager-update"
-  $(npm bin)/webdriver-manager update
+  $(npm bin)/webdriver-manager update --gecko false
```
Comment (IgorMinar) [inline scripts/ci/install.sh]: can you add a comment about the gecko flag?

**Review by IgorMinar — APPROVED** [review]
otherwise lgtm

File: scripts/ci/install.sh (hunk 4 above)
Comment (tbosch) [reply scripts/ci/install.sh]: Done

File: packages/examples/upgrade/static/ts/module.ts [hunk 5]
```diff
@@ -140,7 +140,7 @@ ng1AppModule.factory(
 
 // #docregion downgrade-ng2-heroes-service
 // Register an AngularJS service, whose value is the "downgraded" Angular injectable.
-ng1AppModule.factory('heroesService', downgradeInjectable(HeroesService));
+ng1AppModule.factory('heroesService', downgradeInjectable(HeroesService) as any);
```
Comment (gkalpak) [inline packages/examples/upgrade/static/ts/module.ts]: Why was this necessary? `downgradeInjectable()` returns `Function` and `.factory`'s second argument is of type `Function`?
This shows up in the docs, so we should avoid `as any`.

@tbosch, where were you getting errors without `as any`?
